### DISCUSSION TITLE
Question about Neural Prophet performance

### DISCUSSION THREAD

**Discussion by landkwon94** [ourownstory/neural_prophet#1430] [Q&A - get help using NeuralProphet]
Hello! I asked several times before, and now I am writing manuscripts using Neural Prophet model for cropland monitoring :) 

Thank you so much in advance!

By the way, I have a question about the performance of this model.

In my case, I want to mainly forecast peak season of target variable, using 6 lagged regressors.

Because those peak values are determined by regressor values from the other days.

It seems Neural Prophet model is training the model in the whole days.

So my question is that, is Neural Prophet possible to train in the specific season and forecast in the other specific season?

I attached a figure below for better understanding!

Thank you again for the nice model!

![그림4](https://github.com/ourownstory/neural_prophet/assets/73383617/d114552d-2e30-4b4b-a8a4-6dbbfad0bb29)

**Comment by leoniewgnr** [comment]
Yes this is possible and this is also how it is intended! Very cool use case!
So what you can do, split the dataset into train and test set (either with `m.split_df` or with pandas directly)
And then train on the first set:
`m.fit(df_train)`
and then test/predict on the test set:
`metrics = m.predict(df_test)`

You can see an example in this tutorial: https://neuralprophet.com/tutorials/tutorial10.html

**Comment by leoniewgnr** [comment]
I have just looked more closely on your picture. Do you really wanna train on the blue parts or just use the blue parts to forecast the next red part?

Because if you really want to train the model, you should use cross-validation: https://neuralprophet.com/how-to-guides/feature-guides/test_and_crossvalidate.html

If you only want to forecast, you can just call `m.predict` with `n_forecast` set in the model and you can only predict at a specific point in time (eg always midnight) by setting the `prediction_frequency`: https://github.com/ourownstory/neural_prophet/blob/50d5e4175740c586e9372ffccfcfccc3beac41c1/neuralprophet/forecaster.py#L319

**Reply by poo5297** [reply]
@leoniewgnr I want to evaluate 2 different models that I have tried in Neural Prophet. One is a global local model set with these hyperparameters:
trend_global_local="local",
season_global_local="local",

Second one is a basic model.

While evaluating the test metrics for the global model, I can see a log message:
**WARNING - (NP.forecaster.test) - Note that the metrics are displayed in normalized scale because of local normalization.**

I want to be able to evaluate the test metrics after they are used in the same scale. I would either like to enable normalization of test metrics for model 2 or disable it for model 1. How can I achieve this?
